### PR TITLE
Porting pciutils sanity smoke

### PR DESCRIPTION
**Pull request by ftjahjad**

### CODE REVIEW

**Comment by ftjahjad** [comment]
Hi @veruu , and @rasibley 
Please review the PR.

Thanks,

File: pciutils/sanity-smoke/Makefile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	chmod a+x runtest.sh
+
+clean:
+	rm -f *~ $(BUILT_FILES)
+
+
+include /usr/share/rhts/lib/rhts-make.include
+
+$(METADATA): Makefile
+	@echo "Owner:           Branislav Nater <bnater@redhat.com>" > $(METADATA)
```
Comment (rasibley) [inline pciutils/sanity-smoke/Makefile]: Thanks @ftjahjad, should we change the owner to will here ?

**Comment by rasibley** [comment]
@ftjahjad I left just one comment, otherwise looks good to me :-)

File: pciutils/sanity-smoke/Makefile (hunk 1 above)
Comment (ftjahjad) [reply pciutils/sanity-smoke/Makefile]: @rasibley 
oopsie :+1: . Correct, we should change the owner. 
I will update the owner in Makefile.

**Pull request merged**